Closes #17 display dataset extracts

jeffreyad opened the pull request:
## Pull Request

*DESCRIPTION GOES HERE*

Before you submit your pull request, take a look at the following checklist. Many thanks for your contribution! 

- [ ] **Title:** Place `Closes #<insert_issue_number>` at the beginning of your PR title. Use the Edit button in the top-right if you need to update.
- [ ] **Linked Issue:** Ensure the related issue is linked in the "Development Section" on the right-hand side.
- [ ] **First Contribution:** If this is your first contribution, add yourself to the `DESCRIPTION` file.
- [ ] **Impact on Examples:** If your updates impact any examples, review locally for warnings or errors in the impacted example pages.
- [ ] **Merge Conflicts:** Developers should address any merge conflicts and merge upon successful review.
- [ ] **New Packages:** If new packages were used, ensure they are included in the `DESCRIPTION` file's `Imports` section.
- [ ] **Updated Examples:** If you added or updated an example, ensure it runs on the latest CRAN release versions of all packages used.
- [ ] **Testing Instructions:** Provide instructions on how to test the code if necessary.

bms63 commented:
@Fanny-Gautier @jeffreyad is already to the rescue!!

jeffreyad commented:
I'm finally able to render without errors!

jeffreyad commented:
> @Fanny-Gautier @jeffreyad is already to the rescue!!
> 
> @jeffreyad we were looking at customizing the tables more as they aren't very pleasing as vanilla version of reactable.

Great minds think alike!

rossfarrugia commented on adam/adpc.R:
```diff
@@ -186,6 +195,8 @@ adpc_next <- adpc_prev %>%
     check_type = "none"
   )
 
+reactable(adpc_prev)
```
Any reason examples like this one don't display the very latest dataframe like `adpc_next` in this case?

rossfarrugia reviewed (changes requested):
Thanks @jeffreyad  for the updates! Happy to try reactable tables here for our examples site to hopefully make our articles more presentable. Some comments for consideration:

- Do we need to show so many of these table examples? Maybe just add these when we think there is sufficient value to show the viewer something newly created like an important new variable or parameter. Otherwise it might break up the flow of the article sometimes if there’s too many
- Should we restrict the tables to key columns/records that we want the viewers to pay most attention to?
- Add reactable to DESCRIPTION
- Can you update the other ADaM examples please with the same? For ADSL & ADVS we’ll need to wait til Fanny’s PR merges first

bms63 commented:
@jeffreyad and @rossfarrugia  admiraldiscovery has some really nice code we could use here in the examples site.
https://github.com/pharmaverse/admiraldiscovery/blob/main/R/interactive.R

I propose we make a functions folder and add in the code and make it fit for purpose

rossfarrugia commented:
@jeffreyad Fanny's ADSL and ADVS updates are now merged so go ahead and merge latest main to this branch and please make similar updates for those articles too. I'll be starting ADAE hopefully later this week and i'll follow your example from here.

jeffreyad commented:
> @jeffreyad Fanny's ADSL and ADVS updates are now merged so go ahead and merge latest main to this branch and please make similar updates for those articles too. I'll be starting ADAE hopefully later this week and i'll follow your example from here.

Thanks @rossfarrugia will do

jeffreyad commented:
Hi @rossfarrugia @bms63 @Fanny-Gautier, I added a function that added more styling to the `reactable` tables.  I updated ADSL, ADVS, ADPPK, and ADPC.  I can update ADRS and ADTTE if you like the styling.

rossfarrugia commented:
@jeffreyad looks good from my perspective but i defer to @bms63 to also sanity check as he requested this one!

bms63 commented:
This is looking really nice!! 
![image](https://github.com/user-attachments/assets/1adf2af4-a9d9-44f6-88c4-8e23361e66b3)

bms63 commented:
I think we need to add a footnote to each table saying this is just small subset of the data to view.  Please run the dataset program if you would like to see the entire dataset.  

Something seems off in the MAP, BMi section 
![image](https://github.com/user-attachments/assets/0b709240-2c06-4c8c-b3d6-762055fcde20)


This is such a nice touch - didn't know the website did this 
![image](https://github.com/user-attachments/assets/cd19b22b-5fc2-4e3a-91ea-facaae1f724f)

jeffreyad commented:
Yes, I will add a note about it being a subset of data

rossfarrugia reviewed (approved):
Let's give this a go! I'm going to go ahead and approve/merge and then merge it into the ADAE example. We can always continually improve over time if we spot any more. Thanks for the efforts Jeff!!
